terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -1 +1,25 @@
-# placeholder file
\ No newline at end of file
+environment = "live"
+aws_profile = "live-eu-west-2"
+
+# scaling configs
+desired_task_count = 2 # use multi instance in staging
+
+# service configs
+api_local_url = "https://private-api.companieshouse.gov.uk"
+cdn_host = "//db2062spq951s.cloudfront.net"
+chs_url = "https://find-and-update.company-information.service.gov.uk"
+cookie_domain = "company-information.service.gov.uk"
+piwik_site_id = "10"
+piwik_url = "https://matomo.companieshouse.gov.uk/"
+session_create = "off"
```

Comment:
is the session_create definitely 'off' in live?

Reply:
Yes. Confirmed by config sent to me by Darran Cornelius.

Reply:
ok, i also went through the release notes and can see it was set to off in first release and never changed, so should be good.